### PR TITLE
First draft of mission statement

### PR DESCRIPTION
**Pull request by jessemapel**
This is for discussion at the 7/9 meeting.

### CODE REVIEW

**Review by jlaura — APPROVED** [review]

**Comment by rbeyer** [comment]
As discussed during meeting: perhaps soften the language to talk about how the goal is to provide structure to the extent that it helps, but not just to impose boxes and walls for the purpose of being strict.  Change it to be more about facilitating the external group's work, and get away from implying that order will be imposed.

**Comment by michaelaye** [comment]
I was also wondering if it's clear enough that we also support (advise?) in technical guidance? Or don't we? Guess that could create a helluva job? I'm thinking of answering questions on how to package, how to test, how to autodoc, etc..

File: Mission.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Mission Statement
+
+The Planetary Software Organization promotes open source software in the planetary sciences by helping software creators and maintainers foster an open source community.
+
+The Planetary Software Technical Steering Committee (TSC) helps software creators grow their community by providing community guidelines and standards. The TSC also helps software maintainers manage the growth and evolution of open source software by providing an open source governance model.
```
Comment (SgStapleton) [inline Mission.md]: "Planetary Software Organization Technical..." ?

Comment (SgStapleton) [reply Mission.md]: Just the naming standard as per the README in our repo. I think consistency would be key here.

Comment (rbeyer) [reply Mission.md]: Fair point.

**Comment by victoronline** [comment]
What do you all think about updating this snippet a bit? "helps software creators grow their community by providing community guidelines and standards." to something similar to "helps software creators grow and organize their community by providing a set of ready-to-use community guidelines and standards."?  I'm just trying to emphasize the idea that this the TSC is extending a resource and not "roman soldiering" the project.

**Comment by jessemapel** [comment]
I have some time before the meeting to continue to massage this, but how do people feel about the current text? I think it provides the same information but defines us as supporting the software maintainers and not constraining them.

**Review by rbeyer — APPROVED** [review]
I like these changes.

**Review by michaelaye — APPROVED** [review]

**Review by jlaura — APPROVED** [review]

**Comment by SgStapleton** [comment]
I think these changes are sounding much more polished.

**Pull request merged**